CONTRIBUTING.md

```diff
@@ -100,7 +98,7 @@ Pull requests will be merged based on the following criteria:
 - Does not have the `do-not-merge/hold` label.
 - Does not have the `wip` label.
 - All status checks have succeeded.
-- If the person who opened the pull request is a core maintainer, then only that person is expected to merge once it has the necessary LGTMs/reviews. Another maintainer can merge the pull request at their discretion if they feel the pull request must be merged urgently.
+- Pull requests are automatically merged using the automerge-action bot. More on how OSM uses this bot in the PR queue [here](https://github.com/openservicemesh/osm/blob/main/docs/developer/automerge.md)
```

Comment:
This should be optional when someone wants to merge a pull request. By default review authors should be able to hold PRs as long as they want.